TKeazirian.HTTPServer/Router/Router.cs

```diff
@@ -19,6 +19,11 @@ public Response Route(Request request)
 
         if (RoutesConfigContainsPath(request) && !IsHttpMethodAllowed(request))
         {
+            if (IsOptionsRequest(request))
+            {
+                return new OptionsResponse().BuildOptionsResponse(request, _routesConfig);
+            }
```

Comment:
This looks like it's the same code as lines 37-40. Do you need it in both places?

Reply:
I did notice that duplication, as well!  Working on fix right now!